release-notes.txt

```diff
@@ -719,6 +719,7 @@ There are 190 known issues in this release
 Fixed Issues since the last release (2022.04)
 =============================================
 
+- core: "Invalid read of size 4" (#7199)
```

Comment:
It's now "5 issues fixed" in line 728 ;-).

Reply:
Thanks, fixed.